frontend/src/scenes/pipeline/batch-exports/BatchExportEditForm.tsx

```diff
@@ -476,10 +476,15 @@ export function BatchExportsEditFields({
                 ) : batchExportConfigForm.destination === 'HTTP' ? (
                     <>
                         <LemonField name="url" label="URL">
-                            <LemonInput />
+                            <LemonSelect
+                                options={[
+                                    { value: 'https://us.i.posthog.com', label: 'https://us.i.posthog.com' },
+                                    { value: 'https://eu.i.posthog.com', label: 'https://eu.i.posthog.com' },
```

Comment:
Maybe easier for users to have `PostHog US` and `PostHog EU` as labels?

I'd also clarify in the field label that this is the "Destination URL", just to avoid confusion with the current URL a user may be visiting in case they are doing cross-region migrations.